### ISSUE TITLE
Plugin broken on Wordpress 4.1

### ISSUE THREAD

**Issue by slickdev** [halgatewood/specials#1]
Any way you can figure out what the problem is? The pages are missing CSS styles and images.

**Comment by halgatewood** [comment]
Thanks for writing me. I'll check it out very soon!

**Comment by halgatewood** [comment]
Are you using the basic [specials] shortcode with nothing else?

**Comment by slickdev** [comment]
Yes. I ended up finding a solution. I needed to change the permalink from /specials/ to /the-specials/. For some reason, when I access /specials/, Wordpress loads an archive template. The shortcode never gets rendered. Changing the permalink solved the problem for me.
